### PR TITLE
19.11.1906

### PR DESCRIPTION
**Pull request by are17**
Page 1 Monday
Deliverable 1

### CODE REVIEW

**Comment by are17** [comment]
Are you able to see my latest pull request? I asked Weber for help but not sure if you can see the updated version of my XML file.

File: 1906-11-19.xml [hunk 1]
```diff
@@ -0,0 +1,3685 @@
...
+
+                        <cell>about <date when="1906-11-21">November 21</date></cell>
+
+                        <cell>Belgian Prince</cell>
+
+                        <cell>due from</cell>
+
+                        <cell><placeName>Manchester</placeName></cell>
+
+                        <cell><date when="1906-11-25">November 25</dat
```
Comment (whanley) [inline 1906-11-19.xml]: This story is a new item--add a div tag before it.

**Comment by whanley** [comment]
Adrian, there's lots of good work here, but also some problems. Your divs are not working correctly. Also, there is a problem with pasting your OCR text into Oxygen. @huoyanzhizhou and I can help you with this, but I think we need to find time to sit down together--it should take about 20 minutes.

**Comment by are17** [comment]
I was wondering if we could set up a time to meet, unfortunately due to my schedule I can't make your office hours. Fridays are also hit or miss depending on my practice schedule, so if we could meet I think it would be a huge help. I think if I can just get a better understanding of how to do this in a more time efficient way that would help tremendously. Right now it takes me hours to finish one page and I know there is still more editing for me to do with it.

I appreciate you taking the time to do this for me.

Thank You!

________________________________

**Comment by whanley** [comment]
@are17 if you can make it come to my office (Bellamy 417) at 9 am tomorrow and we can look at things before class.

**Comment by are17** [comment]
I will try to make it to your office at that time. Thank You!

________________________________

**Comment by whanley** [comment]
Wednesday looking quite complete. Make sure that you're filling in the contents of the tables.
Monday divs still need lots of work.

**Pull request merged**